pkg/daemon/controller/utils.go

```diff
@@ -270,7 +270,8 @@ func nodeBelongsToNetwork(nodeName string, network *networkingv1.Network) bool {
 }
 
 func collectGlobalNetworkInfoAndInit(ctx context.Context, client client.Reader, nodeVxlanIfName, nodeName string,
-	bgpManager *bgp.Manager, recordBGPPeers bool) (vxlanForwardNodeIfName string, bgpGatewayIP net.IP, err error) {
+	bgpManager *bgp.Manager, recordBGPPeers bool) (vxlanForwardNodeIfName string, attachedBGPNetworkExist bool,
```

Comment:
the return "attachedBGPNetworkExist" may always be false

Reply:
fixed